setup.py

```diff
@@ -1,21 +1,49 @@
 """Setup script.
 
-Run "python3 setup --help-commands" to list all available commands and their
+Run "python3 setup.py --help-commands" to list all available commands and their
 descriptions.
 """
-import json
+import os
+import shutil
+import sys
 from abc import abstractmethod
 from pathlib import Path
 from subprocess import call, check_call
 
 from setuptools import Command, setup
+from setuptools.command.develop import develop
+from setuptools.command.egg_info import egg_info
+from setuptools.command.install import install
 
+if 'bdist_wheel' in sys.argv:
+    raise RuntimeError("This setup.py does not support wheels")
 
-def read_version_from_json():
-    """Read the NApp version from NApp kytos.json file."""
-    file = Path('kytos.json')
-    metadata = json.loads(file.read_text())
-    return metadata['version']
+# Paths setup with virtualenv detection
+if 'VIRTUAL_ENV' in os.environ:
+    BASE_ENV = Path(os.environ['VIRTUAL_ENV'])
+else:
+    BASE_ENV = Path('/')
+
+NAPP_NAME = 'pathfinder'
+NAPP_VERSION = '2.2.0'
+
+# Kytos var folder
+VAR_PATH = BASE_ENV / 'var' / 'lib' / 'kytos'
+# Path for enabled NApps
+ENABLED_PATH = VAR_PATH / 'napps'
+# Path to install NApps
+INSTALLED_PATH = VAR_PATH / 'napps' / '.installed'
+CURRENT_DIR = Path('.').resolve()
+
+# NApps enabled by default
+CORE_NAPPS = ['of_core']
```

Comment:
Pathfinder install doesn't depend on `of_core`, so you can just leave CORE_NAPPS empty.

Reply:
Updated. Solved in last push.